kore/src/Kore/Step/Axiom/Registry.hs

```diff
@@ -162,10 +162,10 @@ equalitiesToEvaluators
   =
```

Comment:
This function takes multiple rules, and `scatter`s them, so I try to figure out how to keep track of each rule